script_runner.py

```diff
@@ -78,6 +79,9 @@ def _run_command(self, cmd, working_dir):
 
         command_thread.start()
 
+        console_flush_thread = threading.Thread(target=self.__trigger_flush_console_output)
```

Comment:
Not sure I understand why that's a problem... why not just do
````
t = threading.Timer(self.config['TIME_INTERVAL'], self.__trigger_flush_console_output)
t.start()
````
like we do from inside the function